binding/python/docs/EXPERIMENTS.md

```diff
@@ -0,0 +1,60 @@
+# Experiments
+
+Here is the result of running [Deep_Residual_Learning_CIFAR-10](../examples/theano/lasagne/Deep_Residual_Learning_CIFAR-10.py)
+
+## Task Description
+Perform CIFAR-10 classification with residual networks implementation based on Lasagne.
+
+## Hardware
+|||
+| -------- |:--------:|
+|Hosts|1|
+|GPU|GeForce GTX TITAN X * 4|
+|CPU|Intel(R) Core(TM) i7-5960X CPU @ 3.00GHz  * 1|
+|Memory| 128GB |
+
+
+## Theano settings
+Here is the content of `~/.theanorc`
```

Comment:
"Configuration in `~/.theanorc`" or maybe combine with the previous line as something like "## Theano Settings (`~/.theanorc`)"